openssl/src/crypto/rsa.rs

```diff
@@ -42,14 +42,14 @@ impl Drop for RSA {
 impl RSA {
     /// only useful for associating the key material directly with the key, it's safer to use
     /// the supplied load and save methods for DER formatted keys.
-    pub fn from_public_components(n: BigNum, e: BigNum) -> Result<RSA, ErrorStack> {
+    pub fn from_public_components(n: BigNum,
```

Comment:
Oh I have no idea what it is, we just needed to pass something to the API on 1.1.0. I guess that should be null?